yogstation/code/game/gamemodes/shadowling/shadowling.dm

```diff
@@ -0,0 +1,272 @@
+#define LIGHT_HEAL_THRESHOLD 2
+#define LIGHT_DAMAGE_TAKEN 7
+
+#define LIGHT_DAM_THRESHOLD 0.25
+#define LIGHT_RED_MULTIPLIER 0.4
+#define LIGHT_GREEN_MULTIPLIER 0.8
+#define LIGHT_BLUE_MULTIPLIER 1.2
```

Comment:
Why do different colours emit different amounts of light?

Reply:
to spice it up. this'll only affect sling damage in light and how much for the shield to be off, etc